.circleci/config.yml

```diff
@@ -23,35 +23,16 @@ commands:
         failure_message: ':red_circle: M1: A $CIRCLE_JOB job has failed!'
         only_for_branches: master
         webhook: $SLACK_GREENKEEPER_WEBHOOK
-  notify-rc-tag:
-    description: 'Notifies #eng-magento1 when a release candidate is tagged.'
-    parameters:
-      tag:
-        type: string
-        default: ''
-    steps:
-    - slack/notify:
-        channel: "#eng-magento1"
-        color: '#33ff00'
-        mentions: '@oleksii,@Pavel,'
-        message: ":white_check_mark: A new RC << parameters.tag >> has been tagged for Magento 1!"
-        webhook: $SLACK_GREENKEEPER_WEBHOOK
 jobs:
-  tag-rc-candidate:
-    description: Tag master as rc every 3 weeks
+  tag-release-candidate:
     docker:
     - image: cimg/base:2020.01
     steps:
+    - checkout
     - run:
-        name: Validate eligibility to tag rc
-        command: .circleci/scripts/rc-tag.sh
-    - when:
-        condition: $TAGTHISWEEK
-        steps:
-        - swissknife/publish_github_release:
-            tag: $NEWTAGNAME
-        - notify-rc-tag:
-            tag: $NEWTAGNAME
+        name: Tag master as release candidate every 3 weeks
+        command: .circleci/scripts/tag-rc-if-necessary.sh
+    - notify-buildcop
```

Comment:
Will this work? I thought this only notifies on failure

Reply:
yea actually i think i'll just remove this. the script is extremely simple and if it fails it's probably outside of our control (like github outage or something).

Reply:
Removed.